### PR TITLE
Improve documentation and README

### PR DESCRIPTION
**Pull request by nemi-notrace**

### CODE REVIEW

File: REPCO-SPEC.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# repco
+# repco [no longer up to date]
```
Comment (Frando) [inline REPCO-SPEC.md:1]: Let's add a paragraph below the header like

*This specification is currently in review and will be updated soon. Please check the [doc](doc) folder for up-to-date descriptions of the data model.*

or something like that.

**Comment by Frando** [comment]
Let's also rename the files. I propose something like:

```
docs/datamodel.md
docs/datasources.md
docs/repo.md
docs/packages.md
```

or similar

**Comment by nemi-notrace** [comment]
> Let's maybe also rename the files into a simpler structure? I propose something like:
> 
> ```
> docs/datamodel.md
> docs/datasources.md
> docs/repo.md
> docs/packages.md
> ```
> 
> or similar

Changed name and folder struct, also added an extra file for the RLDM

**Comment by Frando** [comment]
I'm going ahead and merging this. It's much better to what we had and we can refine in subsequent PRs.

**Comment by Frando** [comment]
Can you remove the non-markdown changes from the PR? Let's keep this limited to the markdown docs I'd say. Or resolve the conflicts.

**Comment by Frando** [comment]
I merged this in manually. Closing.

**Pull request closed without merging**